ideasbox/templates/index.html

```diff
@@ -3,17 +3,28 @@
 {% load i18n ideasbox_tags %}
 
 {% block content %}
-    <div class="row">
-        <div class="col half">
-            <h2 class="flow"><span class="theme create">blog</span><span>{% trans "LAST POSTS" %}</span></h2>
-            {% for content in blog_contents %}
-                {% include 'blog/content_card.html' %}
-            {% endfor %}
+    <div class="row column_small_screen">
+        <div class="col third">
+            {% include 'ideasbox/includes/wikipedia_card.html' %}
+            {% include 'ideasbox/includes/khan-academy_card.html' %}
+            {% include 'ideasbox/includes/wikipedia_card.html' %}
```

Comment:
Yes, the goal of duplicating was to have the view with more content in three columns for large screen. Indeed, we do not need it for the moment.